src/api/models/users.js

```diff
@@ -58,6 +58,9 @@ var studentSchema = new Schema({
     },
     locations_desired: {
         type: [String]
+    },
+    favorites: {
```

Comment:
You should make a reference to employer and then use .populate() in order to automatically pull in the employers
```
favorites: [{
    type: Schema.Types.ObjectId,
    ref: 'Employer'
}]
```

and then .populate()
```
User.find({ }).populate('favorites').exec(...)
```

Reply:
https://stackoverflow.com/questions/14363065/mongoose-mongodb-query-joins-but-i-come-from-a-sql-background

Reply:
I tried this at first but it broke the comparison logic. Since we're ditching that anyway and letting the client handle it, I'll use this